spec.txt

```diff
@@ -3496,6 +3486,126 @@ can occur one after another, without intervening blank lines.
 ````````````````````````````````
 
 
+The text of a [link reference definition] can be interrupted by any block element
+that can interrupt a paragraph. These are not definitions at all,
+because what would be a multi-line title is interrupted
+by a heading, quoted text, and a fenced code block:
+
+```````````````````````````````` example
+[foo]: /url '
+# h1
+'
+
+[foo]: /url '
+> quote
+'
+
+[foo]: /url '
+```
+text
+```
+'
+
+[foo]
+.
+<p>[foo]: /url '</p>
+<h1>h1</h1>
+<p>'</p>
+<p>[foo]: /url '</p>
+<blockquote>
+<p>quote
+'</p>
+</blockquote>
+<p>[foo]: /url '</p>
+<pre><code>text
+</code></pre>
+<p>'</p>
+<p>[foo]</p>
+````````````````````````````````
+
+
```

Comment:
I thought the spec used one blank line between things, rather than doubles?

Reply:
I am happy to do whatever is the convention. The nearby text seemed to be using doubles.